pkg/plugin/system/system.go

```diff
@@ -251,5 +202,5 @@ func (l *PluginList) insert(p plugin.Plugin) {
 // PluginNode is a node in the PluginList linked list.
 type PluginNode struct {
 	next   *PluginNode
-	plugin plugin.Plugin
+	plugin *client.PluginInstance
```

Comment:
rename to `instance` to avoid `plugin.plugin` in some places? 🤔

Reply:
`payload`